### PR TITLE
tvnamer 2.3 (new formula)

### PR DESCRIPTION
**Pull request by pgilad**

### CODE REVIEW

File: Library/Formula/tvnamer.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    end
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  test do
+    system "#{bin}/tvnamer", "--help"
```
Comment (xu-cheng) [inline Library/Formula/tvnamer.rb]: Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Comment (pgilad) [inline Library/Formula/tvnamer.rb]: Sure, will work on it soon...

**Comment by pgilad** [comment]
@xu-cheng Updated test and rebased branch :smile:

File: Library/Formula/tvnamer.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+class Tvnamer < Formula
+  homepage "https://github.com/dbr/tvnamer"
```
Comment (bfontaine) [inline Library/Formula/tvnamer.rb]: Could you add a short description? The syntax is `desc "the description…"`. Thanks!

**Comment by pgilad** [comment]
@bfontaine Done.

You guys should add that one to the `brew audit` checks

**Comment by xu-cheng** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Pull request closed without merging**